manuscript/05-Other-Service-Mesh.md

```diff
@@ -18,22 +18,22 @@ Kubernetes CRD (Custom Resource Definition) and provides a carefully considered
 
 ## AWS App Mesh {#section-other-aws-app-mesh}
 
 As the name suggests, [AWS App Mesh](http://aws.amazon.com/app-mesh/) is a service mesh developed for AWS. It was first introduced in November 2018. Like Istio and Consul, AWS App Mesh uses a pre-configured but customizable Envoy as service proxy and adds its own control plane. It is integrated with other AWS tools like CloudWatch for metrics and X-Ray for traces. AWS App Mesh already supports essential service mesh features like monitoring and routing. AWS users can activate App Mesh in Fargate, EC2, EKS, and EC2 and add configuration through JSON. In Kubernetes environments, configuration can be applied through CRDs and applied by the [AWS App Mesh Controller](https://github.com/aws/aws-app-mesh-controller-for-k8s).
 
-AWS is the most recent service mesh implementation, but since AWS has the biggest market share for cloud computing, AWS App Mesh is expected to stay and grow.  
+AWS App Mesh is one of the more recent service mesh implementations, but since AWS has the biggest market share for cloud computing, AWS App Mesh is expected to stay and grow.  
 
 ## When to Choose Which? {#section-other-choose}
 
 The features a service mesh provides are useful for the vast majority of microservice applications. When it comes to choosing a specific service mesh implementation, it is hard to resist going with Istio as it is the most popular one. But experience shows that technical decisions are best based on requirements and problems rather than any hype, a project´s public attention, or even its number of features.
 
-As shown in [figure 5.2](#fig-other-service-mesh-comparison), Istio has overtaken all other service mesh implementations in terms of feature completeness and configurability. But this rich feature set has turned Istio into a complex component that can be hard to manage in practice. In cases where not all features and their customizability are required, Linkerd 2, Consul, and AWS App Mesh might be better choices.
+As shown in [figure 5.2](#fig-other-service-mesh-comparison), Istio has overtaken all other service mesh implementations in terms of feature completeness and configurability. But this rich feature set has turned Istio into a complex component that can be challenging to manage in practice. In cases where not all features and their customizability are required, Linkerd 2, Consul, and AWS App Mesh might be better choices.
 
-![Figure 5.2: Features of service mesh implementations as of August 2019](images/other-comparison.png){#fig-other-service-mesh-comparison}
+![Figure 5.2: Features of service mesh implementations as of May 2020](images/other-comparison.jpg){#fig-other-service-mesh-comparison}
```

Comment:
. Probably .png is better suited for the kind of graphics.,

Reply:
The export allows only jpg, so converting makes little sense (right?)